app/renderer/actions/Inspector.js

```diff
@@ -172,19 +152,21 @@ export function unselectHoveredElement (path) {
 export function applyClientMethod (params) {
   return async (dispatch, getState) => {
     const isRecording = params.methodName !== 'quit' &&
-                      params.methodName !== 'source' &&
+                      params.methodName !== 'getPageSource' &&
                       getState().inspector.isRecording;
     try {
       dispatch({type: METHOD_CALL_REQUESTED});
+      const callAction = callClientMethod(params);
       const {contexts, contextsError, currentContext, currentContextError,
              source, screenshot, windowSize, result, sourceError,
              screenshotError, windowSizeError, variableName,
-             variableIndex, strategy, selector} = await callClientMethod(params);
+             variableIndex, strategy, selector} = await callAction(dispatch, getState);
 
       if (isRecording) {
         // Add 'findAndAssign' line of code. Don't do it for arrays though. Arrays already have 'find' expression
         if (strategy && selector && !variableIndex && variableIndex !== 0) {
-          findAndAssign(strategy, selector, variableName, false)(dispatch, getState);
+          const findAction = findAndAssign(strategy, selector, variableName, false);
+          findAction(dispatch, getState);
```

Comment:
should it be awaited?

Reply:
@mykola-mokhnach redux actions don't need to be awaited, you can just fire-and-forget them.

Reply:
yes, i didn't change 'await' status of any of these calls. many of them were intentionally not awaited, they just exist to trigger actions, and there is a sort of event/callback system that happens from there